### PR TITLE
feat:  responsive

### PR DESCRIPTION
**Pull request by PeeyushPrashant**
- hamburger for aside
- responsive NavBar

Screenshot (158)
Screenshot (159)

### CODE REVIEW

File: src/components/Aside/Aside.css [hunk 1]
```diff
@@ -25,21 +25,35 @@
...
+    width: 25rem;
+    height: 100%;
+  }
+
   .sideBar {
     display: none;
   }
+  .show-backdrop {
+    position: fixed;
+    /* top: 7.2rem; */
```
Comment (Rohanmond) [inline src/components/Aside/Aside.css]: remove console

File: src/components/Aside/Aside.css [hunk 2]
```diff
@@ -25,21 +25,35 @@
...
-      background-color: var(--background-color);
-      width: 100%;
-      height: 100%;
-      overflow: hidden;
-    }
-    .sideBar {
-      display: none;
-    }
+@media screen and (max-width: 500px) {
+  /* .show-backdrop {
```
Comment (Rohanmond) [inline src/components/Aside/Aside.css]: remove this comment also

**Review by Rohanmond — APPROVED** [review]
After removing those comments you can go ahead

File: src/components/Aside/Aside.jsx [hunk 3]
```diff
@@ -1,16 +1,19 @@
...
+  const {sideBar, sideBarHandler}= useData();
     return (
-        <aside className="sideBar flex-col"
-        // {sideBar? "showSide flex-col": "sideBar flex-col"}
+      
+      <div className={sideBar? "show-backdrop": "hide-backdrop"}>
+        <aside className={sideBar? "showSide  flex-col": "sideBar flex-col"}
         >
           <NavLink to="/"
         //   style={getActiveStyle}
```
Comment (varnita619) [inline src/components/Aside/Aside.jsx]: remove unwanted comments

File: src/components/NavBar/Navbar.jsx [hunk 4]
```diff
@@ -45,25 +45,25 @@ export const NavBar=()=>{
...
           <i className="fas fa-search search-icon"></i>
-          <input type="text" className="nav-input" placeholder="Type to search"
-          onKeyDown={(e)=>{
-            if(e.key === 'Enter' || e.target.value === ''){
-              filterDispatch({type:"FILTER", payload:["search",e.target.value]})
-            }
-          }}
+          <input type="text" className="nav-input" placeholder="Search for videos"
+          // onKeyDown={(e)=>{
+          //   if(e.key === 'Enter' || e.target.value === ''){
```
Comment (varnita619) [inline src/components/NavBar/Navbar.jsx:59]: remove all comments

**Review by varnita619 — APPROVED** [review]
Consistency in all units
Just remove comments 
Rest all is good

**Review by romabulani — APPROVED** [review]
- Code is clean and easily understandable
- Good to merge

Suggestions : 
- If possible, z-index of sidenav should be greater than that of searchbar, it would look more good
- Remove comments and console not required

File: src/components/Aside/Aside.jsx [hunk 5]
```diff
@@ -39,18 +42,24 @@ export const Aside=()=>{
         <NavLink to="/watchlater"
         // style={getActiveStyle}
```
Comment (vinc3nati) [inline src/components/Aside/Aside.jsx]: always remove commented code before deploying

**Review by vinc3nati — APPROVED** [review]
UI and code looks good, just fix the sidebar bug as it should appear on top of all your content.

**Pull request merged**